### PR TITLE
Add company

### LINKED ISSUE #6: Create Add Company page.

### PR DESCRIPTION
**Pull request by laurandarius**
Closes #6

### CODE REVIEW

**Comment by laurandarius** [comment]
[Create logout "page"](https://trello.com/c/Poqu0kHD/3-create-logout-page)

**Comment by laurandarius** [comment]
[Create add company page](https://trello.com/c/OzeT7egP/4-create-add-company-page)

File: app/containers/AddCompanyForm/actions.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {
+  ADD_COMPANY,
+  ADD_COMPANY_SUCCESS,
+  ADD_COMPANY_FAILED,
```
Comment (lgnstn) [inline app/containers/AddCompanyForm/actions.js]: Please use the noun, "FAILURE", not the verb.

File: app/containers/AddCompanyForm/index.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  },
+  validationSchema: Yup.object().shape({
+    name: Yup.string('')
+      .email('Invalid email address.')
+      .required('This field is required.'),
+    address: Yup.string().required('This field is required'),
+    business: Yup.string().required('This field is required'),
+    rating: Yup.string(),
+    manager: Yup.string(),
+    active: Yup.string(),
```
Comment (lgnstn) [inline app/containers/AddCompanyForm/index.js]: By default a company is active when it is created. This field shouldn't be in the form.

File: app/containers/AddCompanyForm/index.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    active: '',
+  },
+  validationSchema: Yup.object().shape({
+    name: Yup.string('')
+      .email('Invalid email address.')
+      .required('This field is required.'),
+    address: Yup.string().required('This field is required'),
+    business: Yup.string().required('This field is required'),
+    rating: Yup.string(),
+    manager: Yup.string(),
```
Comment (lgnstn) [inline app/containers/AddCompanyForm/index.js]: The user that created the company is the manager, so you can remove this field from the form.

File: app/containers/AddCompanyForm/index.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    manager: '',
+    active: '',
+  },
+  validationSchema: Yup.object().shape({
+    name: Yup.string('')
+      .email('Invalid email address.')
+      .required('This field is required.'),
+    address: Yup.string().required('This field is required'),
+    business: Yup.string().required('This field is required'),
+    rating: Yup.string(),
```
Comment (lgnstn) [inline app/containers/AddCompanyForm/index.js]: The initial rating is 0. This value is going to be changed when the people that use the company's services start to rate the company.

File: app/containers/AddCompanyForm/index.js [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+    name: '',
+    address: '',
+    business: '',
+    rating: '',
+    manager: '',
+    active: '',
+  },
+  validationSchema: Yup.object().shape({
+    name: Yup.string('')
+      .email('Invalid email address.')
```
Comment (lgnstn) [inline app/containers/AddCompanyForm/index.js]: The company name is not an email. Please remove this line.

File: app/containers/AddCompanyForm/index.js [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+      options: {
+        text: 'SUBMIT',
+        name: 'submitButton',
+      },
+    },
+  ],
+};
+
+export class AddCompanyForm extends React.Component {
+  state = {};
```
Comment (lgnstn) [inline app/containers/AddCompanyForm/index.js]: If you don't use the state, don't define it.

File: app/containers/AddCompanyForm/index.js [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+  render() {
+    return <FormComponent {...{ ...config, ...this.props }} />;
+  }
+}
+
+AddCompanyForm.propTypes = {
+  errorMessage: PropTypes.string.isRequired,
```
Comment (lgnstn) [inline app/containers/AddCompanyForm/index.js]: I don't think the error message is a mandatory field. Please remove the `isRequired` property.

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Comment by laurandarius** [comment]
@lgnstn updated.

**Review by lgnstn — APPROVED** [review]

**Pull request merged**